### DISCUSSION TITLE
Does Forme::Form.new accept a Hash of field values as implied by README?

### DISCUSSION THREAD

**Discussion by ScottGrimmett** [jeremyevans/forme#125] [General]
I'm unable to puzzle out the documentation at https://forme.jeremyevans.net that reads:

> If the object doesn’t respond to forme_input, it falls back to creating text fields with the name and id set to the field name and the value set by calling the given method on the object (or using [] if the object is a hash).

```
f = Forme::Form.new([:foo])
f.input(:first) # '<input id="first" name="first" type="text" value="foo"/>'
```
I've pasted the example code into my own project, and verified the output as described. This seems to work because #first is a method of Array. However, the documentation leads me to expect that: 

```
f = Forme::Form.new({foo: "bar"})
f.input(:foo) 
# expected => '<input id="foo" name="foo" type="text" value="bar">'
# actual => '<input type="foo"/>'
```

What am I missing?

**Comment by jeremyevans** [comment]
If you keep reading the README a little longer, in the https://forme.jeremyevans.net/files/README_rdoc.html#label-Forme-3A-3AForm+Creation section, it explains that non-hash arguments are treated differently than hash arguments.  It also explains to use the `:obj` option if you want the form object to be a hash:

```ruby
f = Forme::Form.new(obj: {foo: "bar"})
f.input(:foo)
# => "<input id=\"foo\" name=\"foo\" type=\"text\" value=\"bar\"/>"
```
